docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+   "execution_count": null,
+   "id": "efe16638",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"annotator_stats\"].head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0d09bfa",
+   "metadata": {},
+   "source": [
+    "The `annotator_stats` DataFrame is sorted by increasing `annotator_quality`, showing us the worst annotators first.\n",
+    "\n",
+    "Notice that in the above table annotators with ids 45 to 49 have the worst annotator quality score, which is expected because we made the last 5 annotators systematically worse than the rest."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20ca8dd2",
+   "metadata": {},
+   "source": [
+    "### Comparing improved consensus labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b49657d",
+   "metadata": {},
+   "source": [
+    "We can get the improved consensus labels from the `label_quality_multiannotator` DataFrame shown above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "abd0fb0b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "improved_consensus_label = multiannotator_dict[\"label_quality_multiannotator\"][\"consensus_label\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1fd7a5fd",
+   "metadata": {},
+   "source": [
+    "Since our toy dataset is synthetically generated by adding noise to each annotator's labels, we know the ground truth labels for each example. Hence we can compare the accuracy of the consensus labels obtained using majority vote, and the improved consensus labels obtained using cleanlab."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cdf061df",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "majority_vote_accuracy = np.mean(true_labels == majority_vote_label)\n",
+    "cleanlab_label_accuracy = np.mean(true_labels == improved_consensus_label)\n",
+    "\n",
+    "print(f\"Accuracy of majority vote labels = {majority_vote_accuracy}\")\n",
+    "print(f\"Accuracy of cleanlab consensus labels = {cleanlab_label_accuracy}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c20b2c9",
+   "metadata": {},
+   "source": [
+    "We can see that the accuracy of the consensus labels improved as a result of using cleanlab, which not only takes the annotators' labels into account, but also a model to compute better consensus labels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b806d2ea",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "\n",
+    "if majority_vote_accuracy >= cleanlab_label_accuracy:  # check cleanlab has improved prediction accuracy\n",
+    "    raise Exception(\"Cleanlab training failed to improve model accuracy.\")"
+   ]
```

Comment:
Nice check!

Reply:
Can we also add check that the known low-quality annotators got lower quality scores than the other annotators (eg. take the average of these 5 bad annotators' quality and compare it to avg-quality of the other annotators)